app/models/patient.rb

```diff
@@ -1,3 +1,3 @@
 class Patient < ApplicationRecord
-  validates :name, :birth_date, :city, presence: true
+  validates :name, :birth_date, :city, :medical_history, :surgical_history, presence: true
```

Comment:
I'd say, there's no need of validating neither `medical_history` nor `surgical_history`
I don't have medical history, I don't have surgical history 👀